emukit/core/loop/loop_state.py

```diff
@@ -4,22 +4,6 @@
 
 from .user_function_result import UserFunctionResult
 
-def create_loop_state(x_init: np.ndarray, y_init: np.ndarray) -> None:
```

Comment:
Why did it change place? Not that it matters much, but makes harder to spot what has actually changed

Reply:
The LoopState class needs to be defined for me to use the type annotation without quotes around the class name.

Reply:
Ah, right, makes sense